emoji-selector@maestroschan.fr/prefs.js

```diff
@@ -182,17 +170,22 @@ const EmojiSelectorSettingsWidget = new GObject.Class({
 
 function buildPrefsWidget() {
 	let widget = new EmojiSelectorSettingsWidget();
+	let obj = widget.prefs_stack;
 
-	Mainloop.timeout_add(0, () => {
-		let headerBar = widget.prefs_stack.get_toplevel().get_titlebar();
-		headerBar.custom_title = widget.switcher;
+	obj.connect('realize', () => {
+		let window = (this._shellVersion < 40) ? obj.get_toplevel() : obj.get_root();
 
-		return false;
+		if (this._shellVersion < 40) {
+			headerBar.set_show_close_button(true);
+		}
+
+		this._registerSignals(window);
```

Comment:
@frandieguez **issue:** I have this error:

```
(org.gnome.Shell.Extensions:12542): Gjs-CRITICAL **: 22:59:56.270: JS ERROR: TypeError: this._registerSignals is not a function
buildPrefsWidget/<@/home/stephane/.local/share/gnome-shell/extensions/emoji-selector@maestroschan.fr/prefs.js:184:8
OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:139:20
asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:115:22
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
run@resource:///org/gnome/gjs/modules/script/package.js:206:19
start@resource:///org/gnome/gjs/modules/script/package.js:190:8
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

Reply:
@frandieguez is this a breaking error?

Reply:
It's same for me. Clicking on emojis does nothing. Neither copying them in pastebin nor closing the picker.